cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/FileSetProperties.java

```diff
@@ -50,18 +55,133 @@
    */
   public static final String OUTPUT_PROPERTIES_PREFIX = "output.properties.";
 
+  /**
+   * Whether this dataset should be enabled for explore.
+   */
+  public static final String PROPERTY_EXPLORE_ENABLED = "explore.enabled";
```

Comment:
or if it is meant to reflect explore status, we need to be sure to set accordingly in enable, disable endpoints